packages/cojson/src/coValueState.ts

```diff
@@ -71,12 +69,16 @@ class CoValueUnknownState {
   }
 }
 
-class CoValueAvailableState {
+export class CoValueAvailableState {
   type = "available" as const;
 
   constructor(public coValue: CoValueCore) {}
 }
 
+export class CoValueUnavailableState {
+  type = "unavailable" as const;
+}
```

Comment:
Added this state to mark a CoValue as unavailable.

Used in the load function to see if we already tried to load it and failed.
In that case we try to load the CoValue again. (cc @marinoska this might solve some of the "need to reload" issues)